### PR TITLE
Add althea-rust-binaries to target configs

### PR DESCRIPTION
**Pull request by drozdziak1**
This PR adds the `althea_rs` package to each target's config.

### CODE REVIEW

File: config/virtualbox [hunk 1]
```diff
@@ -23,6 +23,7 @@ CONFIG_OPENSSL_WITH_SRP=y
 CONFIG_PACKAGE_ar=m
 CONFIG_PACKAGE_babel-pinger=m
 CONFIG_PACKAGE_althea-babeld=y
+CONFIG_PACKAGE_althea-rust-binaries=y
```
Comment (jkilpatr) [inline config/virtualbox]: This needs a triple over in the packages repo there's no $ARCH value for x86, unless it works anyways since x86 is the default?

Comment (drozdziak1) [reply config/virtualbox]: The VBox target should help us check this out, I wonder why wouldn't it just say `x86` in `$ARCH`

Comment (jkilpatr) [reply config/virtualbox]: so comments on this? I need to install vbox again and test it properly .

Comment (jkilpatr) [reply config/virtualbox]: sure enough it does need to be handled over in the packages repo. This pull is fine, guess I merged in the wrong order.

**Comment by jkilpatr** [comment]
Well merging this wasn't the best idea. It's not major breakage but still.

I see your point about a consistent CI system @drozdziak1 especially as we start to move fast and do more crossrepo things. I'll have one up as soon as is feasible.

**Comment by drozdziak1** [comment]
Sure, maybe we should revert this until I push the correct x86 stuff through?

**Pull request merged**